cypress/support/pages/help_section/pageHelpSection.js

```diff
@@ -46,25 +42,6 @@ class Help {
 		});
 	}
 
-	clickQuestionIcon() {
-		cy.get(Help.#questionIcon).click();
-	}
-
-	clickHelpSectionInHeader() {
-		cy.get(Help.#helpSectionInHeader).click();
-	}
-
-	clickSendRequestOrProblemInHeader() {
-		cy.get(Help.#sendRequestOrProblemInHeader).click();
-	}
-
-	advancedTrainingsInHeader() {
```

Comment:
method name with verb? eg -> seeAdvancedTrainingsInHeader()

Reply:
This method is removed

Reply:
ah yes sry, its fine.